.vib/spring-cloud-dataflow/goss/spring-cloud-dataflow.yaml

```
@@ -0,0 +1,16 @@
+command:
+  check-spring-cloud-dataflow:
+    # As the version is shown in the output we check it here
+    timeout: 20000
+    exec: timeout --preserve-status 15 /opt/bitnami/java/bin/java -jar /opt/bitnami/spring-cloud-dataflow/spring-cloud-dataflow.jar
```

No need to use the full path here:
```suggestion
    exec: timeout --preserve-status 15 java -jar /opt/bitnami/spring-cloud-dataflow/spring-cloud-dataflow.jar
```
See other examples:
https://github.com/bitnami/containers/blob/96dceffa0d494ea1b75deffa871f8204004ceabe/.vib/cassandra-exporter/goss/cassandra-exporter.yaml#L3
https://github.com/bitnami/containers/blob/96dceffa0d494ea1b75deffa871f8204004ceabe/.vib/prometheus-rsocket-proxy/goss/prometheus-rsocket-proxy.yaml#L9